src/routes/styles.css

```
@@ -1,8 +1,9 @@
 @import '@fontsource/fira-mono';
 
 :root {
-  --font-body: Arial, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu,
-    Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  --font-body:
+    Arial, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
```

Stylelint (17.12.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)